tools/Render1D.py

```diff
@@ -0,0 +1,255 @@
+                        in the case of a single h5 file, it is the file name \
+                        without the .h5 extension')
+    parser.add_argument('--var', type=str, required=True,
+                        help='name of variable to render. eg \'psi\', \
+                        \'psi-Error\' or \'psi-Analytic\'')
+    parser.add_argument('--time', type=int,required=False,
+                        help='if given renders the integer observation step \
+                        instead of an animation (optional)')
+    parser.add_argument('--save', type=str, required=False,
+                        help='set to the name of the output file you want \
+                        written. For animations this saves an mp4 file and \
+                        for stills a pdf. (optional)')
+    parser.add_argument('--fps', type=str, required=False,
+                        help='Set the number of frames per second when writing\
+                        an animation to disk.(optional)')
+    parser.add_argument('--interval', type=str, required=False,
+                        help='number of milliseconds (optional)')
+    parser.add_argument('--list-vars', type=bool, required=False,
+                        help='if true then write list of vars in h5 file \
+                        and exit (optional)')
+    return vars(parser.parse_args())
+
+def get_h5_files(file_prefix):
+    '''
+    Get a list of the h5 files containing the data
+    :return list of h5 files containing volume data
+    '''
+    h5_file_names = glob.glob(file_prefix + "*.h5")
+    h5_files = [h5py.File(file_name, 'r')
+                   for file_name in h5_file_names]
+    return(h5_files)
+
+def print_var_names(args):
+    # This will need to be changed to be consistent with .vol files.
+    '''
+    Print all available variables to screen
+    :param args:
+    :return: None
+    '''
+    file_prefix = args["file_prefix"]
+    h5files = get_h5_files(file_prefix)
+    volfile = h5files[0]["element_data.vol"]
+    obs_id_0  = volfile.keys()[0]
+    grid_names = volfile[Observation_Id_0].keys()
+    grid = volfile[Observation_Id_0][grid_names[0]]
+    variables = grid.keys()
+    variables.remove("connectivity")
+    variables.remove("x-coord")
+    print("Variables in H5 file:\n[%s]" % ', '.join(map(str, variables)))
+    for h5_file in h5files:
+        h5_file.close()
+
+    return None
+
+
+def get_data(args):
+    # This probably will need to be changed in order to
+    # be consitent with the current format of the .vol files.
+    '''
+    Get the data to be plotted
+    #:return: the array of data, coords, and time
+    '''
+    file_prefix = args["file_prefix"]
+    var_name = args["var"]
+    h5files = get_h5_files(file_prefix)
+    volfiles =  [h5file['element_data.vol'] for h5file in h5files]
+    # Get a list of times from the first vol file
+    ids_times = [(obs_id,volfiles[0][obs_id].attrs['observation_value']) for
+                 obs_id in volfiles[0].keys()]
```

Comment:
You need to handle the case where the user makes a typo and there are no matching h5 files.